contrib/babelfishpg_tsql/sql/babelfishpg_tsql.sql

```diff
@@ -2891,3 +2891,20 @@ END;
 $$
 LANGUAGE 'pltsql';
 GRANT ALL ON PROCEDURE sys.sp_sproc_columns_100 TO PUBLIC;
+
+CREATE OR REPLACE PROCEDURE sys.sp_linkedservers()
+AS $$
+BEGIN
+    SELECT 
+    	name AS SRV_NAME, 
```

Comment:
I think case would not be persisted. You will have to enclose the column names in quotes